queue/bytes_queue_test.go

```diff
@@ -40,17 +40,17 @@ func TestReuseAvailableSpace(t *testing.T) {
 	t.Parallel()
 
 	// given
-	queue := NewBytesQueue(21, false)
+	queue := NewBytesQueue(100, false)
 
 	// when
-	queue.Push([]byte("hello1"))
-	queue.Push([]byte("hello2"))
+	queue.Push(blob('a', 70))
+	queue.Push(blob('b', 20))
 	queue.Pop()
-	queue.Push([]byte("hello3"))
+	queue.Push(blob('c', 20))
 
 	// then
-	assert.Equal(t, 21, queue.Capacity())
-	assert.Equal(t, []byte("hello2"), pop(queue))
+	assert.Equal(t, 100, queue.Capacity())
+	assert.Equal(t, blob('b', 20), pop(queue))
 }
 
 func TestAllocateAdditionalSpace(t *testing.T) {
```

Comment:
What do you think about leaving old test untouched (happy path) and add new with more descriptive name (special case)